pkg/utils/errors.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "fmt"
+
+func ErrorIfOccurrs(err error, message string, args ...interface{}) error {
+	if err != nil {
+		return fmt.Errorf(message, args...)
+	}
+
+	return nil
+}
```

Comment:
I'd suggest putting this in an `error` package so call sites would read: `error.IfOccurs`.

Reply:
So then a new directory under utils I guess?

Reply:
Had to call it "errors" otherwise it conflicts with the error builtin type.
So it'll be errors.IfOccurs if that makes sense :)

Another option is errutil or something but I don't think it'll look nicer..